cli/cmdargs/parser.go

```diff
@@ -60,6 +61,7 @@ const (
    following logic:
     - anything that can be interpreted as a decimal integer gets
       an 'int' type
+	- 'nil' string gets 'Any' parameter type and nil value
```

Comment:
Do we have correct indentation here? Also, types mentioned in this section are strings described above (bool/hash160/etc), we don't have 'Any' there, so either NEP-14 should be mentioned, or we describe it in a different way.

Reply:
Fixed.